client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,346 @@
+/**
+ * External dependencies
+ */
+import {
+	applyTestFiltersToPlansList,
+	findPlansKeys,
+	getPlan as getPlanFromKey,
+	getYearlyPlanByMonthly,
+	isMonthly,
+	TERM_MONTHLY,
+	TERM_ANNUALLY,
+} from '@automattic/calypso-products';
+import { Button } from '@automattic/components';
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import PlanPrice from 'calypso/my-sites/plan-price';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getPlan, getPlanBySlug, getPlanRawPrice } from 'calypso/state/plans/selectors';
+
+/**
+ * Internal dependencies
+ */
+import './tabbed-plans-style.scss';
+
+function TabbedPlans( { currencyCode, onUpgradeClick, planProperties } ) {
+	const translate = useTranslate();
+	const tabList = [ 'Limited', 'Professional' ];
+
+	const bothPlansData = {
+		Professional: [
+			translate( 'Unlimited posts/pages/products' ),
+			translate( 'World class 24/7 support' ),
+			translate( 'Custom domain (free for first year)' ),
+			translate( 'WordPress.com ads removed' ),
+			translate( 'Ability to sell products/services' ),
+			translate( 'Earn ad revenue' ),
+			translate( 'Video hosting' ),
+			translate( 'Google Analytics' ),
+			translate( 'MailChimp integration' ),
+		],
+		Limited: [
+			translate( 'Both plans feature 1' ),
+			translate( 'Both plans feature 2' ),
+			translate( 'Both plans feature 3' ),
+			translate( 'Both plans feature 4' ),
+			translate( 'Both plans feature 5' ),
+		],
+	};
+
+	const featureComparisonData = {
+		Professional: [
+			{
+				featureName: translate( 'Install plugins' ),
+				planOne: translate( 'None' ),
+				planTwo: translate( '58,000+ Available' ),
+			},
+			{
+				featureName: translate( 'Themes available' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( ' Available Themes' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/tabbed-plan-signup-test)